### PR TITLE
Completed css-layout-classes exercise. | Parker Siu

### PR DESCRIPTION
**Pull request by parkersiu**
Kapture 2022-10-13 at 13 38 33

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-two-thirds">
+          <h2>Donkey Kong</h2>
+          <p>Donkey Kong is a series of video games featuring the adventures of a gorilla called Donkey Kong,
+            conceived by Shigeru Miyamoto in 1981. The franchise consists of mainly two game genres, but also
+            includes additional spin-off titles of various genres.
+          </p>
+          <div></div>
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:55]: make sure to remove this empty element

File: css-layout-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-one-fourth {
+  flex-basis: calc(100% / 4);
+}
+
+.four {
+  max-width: 100%;
```
Comment (Zhiwutian) [inline css-layout-classes/styles.css:27]: max width is a limiting property, not a set property. you want the images to be `100%` of the width of their parents, and this needs to apply to all images.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking good Parker! Just a couple small updates are needed.

**Comment by parkersiu** [comment]
> Looking good Parker! Just a couple small updates are needed.

Changes have been updated, thanks.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**